tests/test_time_weighted_levels.py

```diff
@@ -0,0 +1,38 @@
+import pyfar as pf
+import numpy as np
+
+def test_time_weighted_levels_decay():
```

Comment:
Could be parametrized to reduce code redundancy
```suggestion
@pytest.mark.parametrize(['time_weighting', 'decay'], [('fast', -34.7), ('slow', -4.3)])
def test_time_weighted_levels_decay(time_weighting, decay):
```